docker/nginx/nginx.conf

```diff
@@ -40,7 +40,10 @@ http {
         '"$http_user_agent" $upstream_response_time '
         '$upstream_bytes_sent $upstream_bytes_received '
         '"$upstream_http_content_type" "$upstream_cache_status" '
-        '"$portal_domain" "$upstream_http_skynet_skylink"';
+        '"$portal_domain" "$upstream_http_skynet_skylink" '
+        'request_time=$request_time '
```

Comment:
Hm well I see why that is super useful, but I'm not sure whether we should be doing it.
First of all it's going to bloat the access logs quite a bit, and second of all it's kinda unfortunate to mix and match the format. I'd say it's either everything or nothing.

Reply:
Great, we're adding these though!

Reply:
If you all feel fine just hit me up on Discord and I'll approve, just don't want to click approve yet without at least discussing whether or not we should add these fields with the descriptive naming thing.